schemas/EGA.ArrayExperiment.json

```diff
@@ -116,52 +128,61 @@
+            "enum": ["DNA assay", "RNA assay", "Metabolomic profiling", "Protein assay"],
+            "example": "DNA assay",
+            "term-curie-pairs": ["DNA assay__EFO:0001456", "RNA assay__EFO:0001457", "Metabolomic profiling__EFO:0000752", "Protein assay__EFO:0001458"]
           },
           "assayed_molecule_curie": {
             "type": "string",
             "title": "Array type of the ArrayExperiment - CURIE",
             "description": "The CURIE (i.e. ontologized term - e.g. EFO:0001456), chosen from a list of CVs, of the assayed molecule.",
-            "enum": ["EFO:0001456", "EFO:0001457", "EFO:0000752", "EFO:0001458"]
+            "enum": ["EFO:0001456", "EFO:0001457", "EFO:0000752", "EFO:0001458"],
+            "example": "EFO:0001456"
           }
         }
       },
       "ADF_files": {
         "type": "array",
         "title": "Array Design Format (ADF) [NCIT:C172213] file block",
         "description": "The array design format (ADF) [NCIT:C172213] is the unique set of probes (with their coordinates) found on the microarray chip. They can be standard (sold by a company) or custom. Its format is of a spreadsheet-like tab-delimited text file with metadata header rows, followed by a multi-column table of probe information. This object shall only be used if the EGA is the one storing such file. Otherwise, the Relationships object shall be used to point from the ArrayExperiment to the ADF submitted elsewhere. #! Using an empty item list but defining the file object as possible additionalItems we create the correct constraint: anything but a file object is rejected, but EGA can add as many as required.",
+        "minItems": 1,
         "additionalProperties": false,
         "additionalItems": { "$ref": "https://github.com/EbiEga/ega-metadata-schema/tree/main/schemas/EGA.common-definitions.json#/definitions/file_object" },
         "items": []
       },
+
       "sample_number": {
         "type": "integer",
         "title": "Number of samples of the ArrayExperiment",
-        "description": "Number of samples included in the experiment. One sample corresponds to one biological replicate [EFO:0002091] (it could be the genetic content from a single cell, a tissue… from a single individual or from several individuals). Shall not be mistaken for technical replicates [CHEBI:24432] being used several times (see https://www.ebi.ac.uk/seqdb/confluence/pages/viewpage.action?spaceKey=EGA&title=Sample+Representation)."
+        "description": "Number of samples included in the experiment. One sample corresponds to one biological replicate [EFO:0002091] (it could be the genetic content from a single cell, a tissue… from a single individual or from several individuals). Shall not be mistaken for technical replicates [CHEBI:24432] being used several times (see https://www.ebi.ac.uk/seqdb/confluence/pages/viewpage.action?spaceKey=EGA&title=Sample+Representation).",
+        "example": "30"
```

Comment:
```suggestion
        "example": 30
```

Reply:
If this is type `integer`, then the example shouldn't be a string

Reply:
Technically true, albeit it will be interpreted as a string, since it's a keyword that we just made up for the future markdown.